scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -5,126 +5,145 @@ import org.w3c.dom.Node
 import org.w3c.dom.NodeList
 import java.io.File
 import java.io.FileInputStream
-import java.util.stream.Collectors
-import java.util.stream.IntStream
 import javax.xml.parsers.DocumentBuilderFactory
 
 /**
  * Script for ensuring that all the Activities in the codebase are defined with accessibility
  * labels.
  *
  * Usage:
- *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_app_level
- *   _manifest_file>
+ *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_manifest
+ *   _files>
  *
  * Arguments:
  * - path_to_directory_root: directory path to the root of the Oppia Android repository.
- * - path_to_app_level_manifest_file: path to the manifest file of app layer.
+ * - path_to_manifest_files: path to the manifest files
  *
  * Example:
  *   bazel run //scripts:accessibility_label_check -- $(pwd) app/src/main/AndroidManifest.xml
  */
 fun main(vararg args: String) {
   val repoPath = "${args[0]}/"
 
-  val accessibilityLabelExemptiontextProto = "scripts/assets/accessibility_label_exemptions"
+  val accessibilityLabelExemptionTextProto = "scripts/assets/accessibility_label_exemptions"
```

Comment:
```suggestion
  val accessibilityLabelExemptionTextProtoFilePath = "scripts/assets/accessibility_label_exemptions"
```

Ditto elsewhere this same name is used. The name wasn't indicating what it contained before--I missed this in the last pass.

Reply:
Done.